sql_generators/glean_usage/templates/event_monitoring_live_v1.materialized_view.sql

```diff
@@ -1,10 +1,14 @@
 CREATE MATERIALIZED VIEW
 IF
   NOT EXISTS `{{ project_id }}.{{ derived_dataset }}.event_monitoring_live_v1`
+  PARTITION BY DATE(submission_date_timestamp)
+  CLUSTER BY channel, event_category
```

Comment:
A couple clustering suggestions:
- Also cluster on `event_name`.
- If you keep separate `submission_date` and `submission_date_timestamp` columns and either don't implement my idea of refactoring the view query to establish a direct relationship between those columns or that turns out not to help with partition elimination, then I'd suggest adding `submission_date` as the first clustering column to help with query performance when filtering on `submission_date`.